src/gwt/src/org/rstudio/studio/client/rsconnect/ui/NewRSConnectAuthPage.java

```diff
@@ -305,14 +305,17 @@ public void execute(NewRSConnectAccountResult input)
                   contents_.showWaiting();
                   
                   // prepare a new window with the auth URL loaded
-                  NewWindowOptions options = new NewWindowOptions();
-                  options.setName(AUTH_WINDOW_NAME);
-                  options.setAllowExternalNavigation(true);
-                  options.setShowDesktopToolbar(false);
-                  display_.openWebMinimalWindow(
-                        result_.getPreAuthToken().getClaimUrl(),
-                        false, 
-                        700, 800, options);
+                  if (canSpawnAuthenticationWindow())
```

Comment:
Yeah, this should probably be more like:

Please click the link below to confirm your account: